shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-scaling/src/test/java/org/apache/shardingsphere/integration/data/pipeline/util/ResultSetUtil.java

```diff
@@ -15,18 +15,26 @@
  * limitations under the License.
  */
 
-package org.apache.shardingsphere.integration.data.pipeline.cases.entity;
+package org.apache.shardingsphere.integration.data.pipeline.util;
 
-import lombok.Data;
-import lombok.RequiredArgsConstructor;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.LinkedList;
+import java.util.List;
 
-@RequiredArgsConstructor
-@Data
-public final class JdbcInfoEntity {
+public final class ResultSetUtil {
     
-    private final String username;
-    
-    private final String password;
-    
-    private final int port;
+    /**
+     * Get first column value from result set.
+     * @param resultSet result set
+     * @return first column value list
+     * @throws SQLException sql exception
+     */
+    public static List<String> getFirstColumnValueFromResult(final ResultSet resultSet) throws SQLException {
+        List<String> result = new LinkedList<>();
```

Comment:
`getFirstColumnValueFromResult` is not common enough, and it is just used in one class.
Could we just put it in used class?